### DISCUSSION TITLE
NET::ERR_CERT_DATE_INVALID

### DISCUSSION THREAD

**Discussion by jge162** [vercel/community#4392] [Help] [answered]
### Summary

My website that is being hosted on Vercel is now showing that the SSL certificate expired on 10/03/2023 and I have removed the domain from vercel, reentered it and push a commit to the github repo and the SSL certicate is still not updated. Shouldn't vercel update the certificate? thanks.

### Example

https://www.ectechnologyservices.com/

### Steps to Reproduce

Just load the webpage and you will see the error below.
<img width="1192" alt="Screenshot 2023-10-06 at 1 26 32 PM" src="https://github.com/vercel/community/assets/31228460/746e71eb-e904-4866-accb-e6f629a9ce5d">

**Comment by jge162** [comment, marked as answer]
Figured out the issue

**Reply by jreoka** [reply]
And the issue was?

**Reply by jge162** [reply]
was trying to redirect to www. Instead of just using https://
